### ISSUE TITLE
ChatBox sizing issue

### ISSUE THREAD

**Issue by nicolas-rabault** [holoviz/panel#5236]
#### ALL software version info
(this library, plus any other relevant software, e.g. bokeh, python, notebook, OS, browser, etc)
panel :  1.2.0
bokeh: 3.2.0
Python 3.9
on macOS

#### Description of expected behavior and the observed behavior
I experience some issues with the sizing of the ChatBox widget. Text may be not aligned with the text bubbles.

#### Complete, minimal, self-contained example code that reproduces the issue

```
import panel as pn

pn.extension(template="material",theme = 'dark')

output = pn.widgets.ChatBox(
    value=[
        {"Bot": "Ask me anything!"},
    ],
)
output.extend([{"You": "What is the meaning of life?"}])

side_box = pn.WidgetBox(
    pn.pane.Markdown("sidebox"),
    height=500,
)

pn.Row(output,thought_box).servable()

```

#### Stack traceback and/or browser JavaScript console output
No text error

#### Screenshots or screencasts of the bug in action
![Screenshot 2023-07-09 at 19 18 06](https://github.com/holoviz/panel/assets/9554146/c3e7255e-fb7c-4efa-abcd-06bc3db48752)

**Comment by ahuang11** [comment]
Thanks for reporting! What browser are you using? I can't seem to reproduce.

<img width="1296" alt="image" src="https://github.com/holoviz/panel/assets/15331990/e2e6180a-3c4e-4267-a9cf-6d15ecb469e0">

**Comment by nicolas-rabault** [comment]
You are right I wasn't able to reproduce it either with my code (my code example was buggy).
I modified it and you should be able to reproduce it now.
I can reproduce it in VScode notebook and chrome.
